arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1082,6 +1082,9 @@
             "$ref": "#/definitions/EventHubProperties"
           }
         },
+        "routing": {
```

Comment:
Adding properties (even optional ones) to a resource types requires a new API version per the Azure Resource Providers versioning guidelines.